src/box/sql.c

```diff
@@ -778,7 +778,25 @@ tarantoolsqlIdxKeyCompare(struct BtCursor *cursor,
 	if (key != NULL) {
 		int new_rc = sqlVdbeRecordCompareMsgpack(key, unpacked);
 		region_truncate(&fiber()->gc, original_size);
-		assert(rc == new_rc);
+		/*
+		 * Here we compare two results from memcmp() alike
+		 * calls. A particular implementation depends on
+		 * a type of msgpack values to compare. For some
+		 * of them we actually call memcmp().
+		 *
+		 * memcmp() only guarantees that a result will be
+		 * less than zero, zero or more than zero. It
+		 * DOES NOT guarantee that the result will be
+		 * substraction of first non equal bytes or
```

Comment:
1. substraction -> subtraction
2. first -> the first
3. non equal -> non-equal

Reply:
Thanks! Updated.